src/scroll-timeline-css.js

```diff
@@ -32,7 +32,7 @@ function initMutationObserver() {
    */
   function handleStyleTag(el) {
     // Don’t touch empty style tags.
-    if (el.innerHTML.trim().length === 0) {
+    if (el.innerHTML.trim().length === 0 || el.dataset.aphrodite === 'true') {
```

Comment:
This doesn't seem correct, in the [aphrodite code](https://github.com/Khan/aphrodite/blob/225f43c5802259a9e042b384a1f4f2e5b48094ea/src/inject.js#L38) they call
```js
styleTag.setAttribute("data-aphrodite", "");
```

This results in having the key set to the empty string so it does not evaluate equivalent to 'true'. I see the same issue on this [demo page](https://codesandbox.io/p/sandbox/css-in-js-demo-aphrodite-65bcs?file=%2Fsrc%2Findex.js) where if i get the style tag this still evaluates false.

Reply:
Let me test our instance again, I feel like I had tested this before. Perhaps both or maybe I got it wrong, will update this Tuesday based on more tests in production.

Reply:
Interesting, yeah I just re-tested and looked at a messaging URL we use and I get the following which is why it is working for us. Not sure if it's different build tools / settings or what, but maybe we can test for any existence of `data-aphrodite` and ignore it by default to avoid having to have a big list or special values.

```html
<style data-aphrodite="true" id="aphrodite" type="text/css">._1k81gub{position:re...</style>
```

Reply:
view-source:https://www.airbnb.com/ for example

Reply:
Updated to only check existence now

Reply:
Checking the existence sounds good to me. I don't know if the value "false" needs to be specifically handled - seems unlikely?

Reply:
Seems unlikely to me as well, probably not worth the added complexity.